src/full/Agda/Interaction/BasicOps.hs

```diff
@@ -1132,8 +1135,20 @@ introTactic pmLambda ii = do
       r <- splitLast CoInductive tel pat
       case r of
         Left err -> return []
-        Right cov ->
-           mapM showUnambiguousConName $ concatMap (conName . scPats) $ splitClauses cov
+        Right cov -> mapM conWithArgs $ catMaybes $ map (conName . scPats) $ splitClauses cov
+      where
+        conWithArgs ch = do
+             cn <- showUnambiguousConName ch
+             sia <- showImplicitArguments
+             return $ unwords $
+                         cn : [ case v of
+                                    Hidden ->      "{?}"
+                                    Instance _ -> "{{?}}"
```

Comment:
I think the `Instance` case should result in omitting the argument.

Reply:
I disagree, but it I guess it depends on what the use case is for inserting the implicit arguments in the first place. I think it would be a lot more useful if the implicit were inserted with their names, so you can delete the ones you don't want.

Reply:
I left instance arguments (if showing implicit args is enabled) for now (they can be easily switched off).
To add the names of the arguments, I switched to inspecting the type of constructor, instead of pattern coming from splitClauses. I am not sure if I did it correctly.

Reply:
The problem with instance arguments is that instance resolution does not kick in for a meta of the form `{{?}}`.  Note that unification _does_ kick in for a meta of the form `{?}`, leaving a constraint if successful.  I think the same mechanism would make sense for instance arguments, for uniformity.

Reply:
In the current version, first, it tries to type check the constructor applied to everything apart from instance arguments against the type of the hole (so instance resolution has the opportunity to kick in).
If this fails it repeats this with but also with metas in instance arguments.
In both cases, it later checks `MetaInstantiation` for each `Elim`, so it can render only arguments where resolution failed or TC left open meta. 
I plan to add tests, and more comments to the code tomorrow morning, and then I will mark PR as ready for review.